extensions/wikia/Hydralytics/css/hydralytics.scss

```diff
@@ -28,10 +28,9 @@ $green-navy: #002A32;
 @media screen and (max-width: 800px) {
 	#analytics_grid { 
 		grid-template-columns: 1fr;
-		grid-template-rows: auto auto auto auto auto auto auto auto auto auto auto auto auto;
+		grid-template-rows: auto auto auto auto auto auto auto auto auto auto auto;
```

Comment:
just auto auto auto auto scss things I guess

Reply:
I am actually unfamiliar with grid css, I just removed those that seems unnecessary